### PR TITLE
Tests: Added tests for `removeSampleOrganization.ts`

### LINKED ISSUE #2079: Test: Create tests for src/resolvers/Mutation/removeSampleOrganization.ts
We will need unittests done for all methods, classes and functions found in this file.

Any widgets, components, modals referenced in this file must also have unittests done

### PR Acceptance Criteria

- When complete this file must show at least **100%** coverage when merged into the code base. 
- [The current code coverage for the file can be found here](https://app.codecov.io/gh/PalisadoesFoundation/talawa-api?search=&displayType=list)
- The PR will show a report for the code coverage for the file you have added. You can use that as a guide
- Please coordinate **issue assignment** and **PR reviews** with the contributors listed in this issue https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by akhilender-bongirwar**
**What kind of change does this PR introduce?**
This PR makes sure `removeSampleOrganization.ts` has 100% code coverage.

**Issue Number:**

Fixes #2079 

**Did you add tests for your changes?**
Yes

**Snapshots/Videos:**
image
image

**If relevant, did you update the documentation?**
N/A

**Summary**
- Removed the test which is skipped
- Added test to ensure admins can removeSampleOrganization without an error.
- Ensured 100% code coverage.
- Removed a redundant test case.

**Does this PR introduce a breaking change?**
No

**Other information**

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-api/blob/master/CONTRIBUTING.md)?**
Yes

### CODE REVIEW

**Comment by palisadoes** [comment]
You have removed tests to get the coverage higher. Why?

**Comment by akhilender-bongirwar** [comment]
@palisadoes, I just removed the duplicate test as these both do the same job. 
```
 it("should NOT throw error when organization doesn't exist", async () => {
    const { requestContext } = await import("../../../src/libraries");
    vi.spyOn(requestContext, "translate").mockImplementation(
      (message) => message,
    );

    const userData = await generateUserData(
      ORGANIZATION_ID.toString(),
      "ADMIN",
    );
    const admin = userData.user;

    const args = {};
    const adminContext = { userId: admin._id };
    const parent = {};

    await SampleData.deleteMany({ collectionName: "Organization" });

    try {
      await removeSampleOrganization!(parent, args, adminContext);
    } catch (error) {
      expect((error as Error).message).toBe(
        ORGANIZATION_NOT_FOUND_ERROR.MESSAGE,
      );
    }
  });

  it("should NOT throw error when organization doesn't exist", async () => {
    const { requestContext } = await import("../../../src/libraries");
    vi.spyOn(requestContext, "translate").mockImplementation(
      (message) => message,
    );

    const userData = await generateUserData(
      ORGANIZATION_ID.toString(),
      "ADMIN",
    );
    const admin = userData.user;

    const args = {};
    const adminContext = { userId: admin._id };
    const parent = {};

    await SampleData.deleteMany({ collectionName: "Organization" });

    try {
      await removeSampleOrganization!(parent, args, adminContext);
    } catch (error: any) {
      expect(error.message).toBe(ORGANIZATION_NOT_FOUND_ERROR.MESSAGE);
    }
  });
  ```
  So, I removed one of these.

File: tests/resolvers/Mutation/removeSampleOrganization.spec.ts [hunk 1]
```diff
@@ -69,57 +69,24 @@ afterAll(async () => {
 });
 
 describe("Remove Sample Organization Resolver - User Authorization", async () => {
-  it.skip("should NOT throw error when user is ADMIN", async () => {
```
Comment (akhilender-bongirwar) [inline tests/resolvers/Mutation/removeSampleOrganization.spec.ts:72]: And previously this test use to get skipped as it was resulting in error. So I modified the test to perform the expected functionality.

**Comment by palisadoes** [comment]
Two issues across all repos. Sometimes we make exceptions, sometimes we forget to check.

**Review by palisadoes — APPROVED** [review]

**Pull request merged**